build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e # Stop script on error
+set -a # Automatically export all variables
+
+#install node
+PWD_NOW=$PWD
+cd /root
+wget https://nodejs.org/dist/v20.15.1/node-v20.15.1-linux-x64.tar.gz    
+tar zxvf  node-v20.15.1-linux-x64.tar.gz                         
+node-v20.15.1-linux-x64/bin/node -v                               
+
+ln -s /root/node-v20.15.1-linux-x64/bin/npm   /usr/local/bin/ 
+ln -s /root/node-v20.15.1-linux-x64/bin/node   /usr/local/bin/
+cd $PWD_NOW
+echo "nodejs install success !"
+
+#install docker, docker-comp
+curl -fsSL https://test.docker.com -o test-docker.sh
```

Comment:
As discussed within the, "Overview," of this pull request, we can use something like Digital Ocean, which provides a pre-installed Droplet that has Docker already installed and managed.

https://github.com/openpredictionmarkets/socialpredict/blob/main/README/DEV_DEPLOY.md

@donghj2000 If we were talking about deploying to a service provider specific to China, perhaps you already are well aware of the availabilities. Through a quick search, I see Alibaba Cloud gives directions on how to do so, though could not find a One-Click install:

https://www.alibabacloud.com/help/en/ecs/use-cases/install-and-use-docker-on-a-linux-ecs-instance

...

From this comment here, Digital Ocean seems slow in Mainland China. https://www.digitalocean.com/community/questions/cover-mainland-china-usa-and-europe

However, this competitor, LightNode, offers a Hong Kong VPS so I am not sure if that would help make it faster. Perhaps you are already well aware of a solution.

https://go.lightnode.com/hong-kong-vps

Reply:
You are right. But I did'n ues any cloud services, I used VPS only such as hostvds, vultr.